github.css

```diff
@@ -0,0 +1,99 @@
+/*
+
+github.com style (c) Vasily Polovnyov <vast@whiteants.net>
```

Comment:
The change is using a js lib (https://github.com/isagalaev/highlight.js) suggested by original post of issue 86. It is using BSD license. 

Does the Selenium docs project require authoring group of any 3rd party open source libs to sign same contributor license agreement? I'm thinking that maybe that's why the doc.js and all html files are written from scratch.

Please kindly let me know and I will close the request for that. Thank you.